src/neynar-api/neynar-api-client.ts

```diff
@@ -2249,6 +2250,66 @@ export class NeynarAPIClient {
     return await this.clients.v2.fetchRelevantFollowers(targetFid, viewerFid);
   }
 
+  /** 
+  * Returns a list of followers for a specific FID.
+  * @summary Retrieve followers for a given user
+  * @param {number} fid User who's profile you are looking at
+  * @param {Object} [options] - Optional parameters for customizing the response.
+  * @param {number} [options.viewerFid] Viewer who's looking at the profile.
+  * @param {FollowSortType} [options.sortType] Sort type for retrieving followers. Default is FollowSortType.DescChron
+  * @param {number} [options.limit] Number of results to retrieve (default 20, max 100)
+  * @param {string} [options.cursor] Pagination cursor.
+  *  omit this parameter for the initial request.
+  * 
+  * @returns {Promise<UsersResponse>} A promise that resolves to a `UsersResponse` object,
+  * 
+  * @example
+  * // Example: Retrieve followers for a user
+  * import { FollowSortType } from "@neynar/nodejs-sdk";
```

Comment:
`FollowSortType` need to be exported from `src/neynar-api/index.ts` in order to import it like this

Reply:
done!